lib/api/core/notifier.js

```diff
@@ -161,26 +154,16 @@ class NotifierController {
   notifyDocumentCreate (request, newDocument) {
     const
       cachePrefix = getCachePrefix(request),
-      rooms = this.kuzzle.realtime.test(
-        request.input.resource.index,
-        request.input.resource.collection,
-        newDocument._source,
-        newDocument._id
-      );
+      rooms = this._test(request, newDocument._source, newDocument._id);
 
     return this
-      .notifyDocument(
-        rooms,
-        request,
-        'in',
-        'done',
-        'create',
-        {
-          _source: newDocument._source,
-          _id: newDocument._id
-        })
+      .notifyDocument(rooms, request, 'in', 'create', {
```

Comment:
I also prefer this way to wrap method with an object as last argument :)

Reply:
I'm still playing with some styles, but this one seems neat. I'm considering adding that rule to our coding style documentation.